index.html

```diff
@@ -41,6 +41,9 @@
         font-size: 13px;
         height: 800px;
       }
+      pre.zenuml {
```

Comment:
Why do we use a different solution here instead of the inline style attribute as in `embed.html`?